code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -72,14 +76,61 @@
 	if(A.totalStageSpeed() >= 6)
 		power = 2
 
+/datum/symptom/heal/starlight/proc/CanTileHeal(turf/T)
+	var/turf/original_turf = T
+	var/area/area_to_check = get_area(T)
+	var/levels_of_glass = 0 // Since starlight condensation only works 2 tiles to the side anyways, it shouldn't work with like 100 z-levels of glass
+	var/current_heal_level = STARLIGHT_CANNOT_HEAL
+	while(levels_of_glass <= 2)
```

Comment:
What's the significance of this number, is it `STARLIGHT_CAN_HEAL` or is that just a coincidence?

Reply:
Its just a coincidence, it is the range that starlight condensation could heal to begin with, so I should probably move it to a define